issues/meaningless_work.md

```diff
@@ -0,0 +1,55 @@
+- architecture
+---
+
+# Meaningless Work
+
+## What Is a Meaningless Work
+
+Work can become meaningless if your team reaches one of these two phases:
+1. The product loses its business value (or it never had any). It does not make any profit, no real goals were met.
+2. The team thinks that the work is meaningless. Team members do not feel a sense of purpose even though the product can be valuable.
+
+It is important for a Scrum Team to deliver value frequently. Product Owners are responsible for maximizing the business value and that the team is reaching goals every sprint. There is no universal definition of value, it all depends on the context. The value is usually calculated for each user story. However, hard workers and high-quality product functions do not always ensure that the product is valuable enough for the company. It is easy to work on the “wrong stuff at the wrong time”.
+
+On the other hand, the feeling of a meaningless work is something that the team can overcome. What makes the team think that the work is meaningless? The list is huge: excessive emails, pointless meetings, constant interruptions, boss’ unreasonable behavior, meaningless activities, lack of collaboration, or ["Not My Problem" mentality](/issues/not-my-problem-mentality) in the team.
+
+The management should be aware of how the employees feel. If they are not motivated and think that their work is meaningless, the management should do some changes. It could be an anonymous evaluation, experimenting with removing some activities, or monitoring the employees’ performances. For some more tips on how to “unstuck” the team, see [Liberationist: A Simple Exercise That Will Unstuck Your Team Immediately](https://blog.liberationist.org/a-simple-exercise-that-will-unstuck-your-team-immediately-d294d5ad1bc7).
+
+## Reasons for the Meaningless Work
+
+- The product does not have customers. Typial for startups without validated assignment or prototyping. There are plenty of techniques for the validation, such as [Lean Canvas](/practices/lean-canvas).
+- Neverending meetings or useless reports that consume time can be cut down by regular rituals (see [Rituals Definition](/practices/rituals-definition)). The goal is to find a balance between havig too many and too little rituals.
+- Another reason is a lack of information. The developers do not understand why they have implement this thing. They did not get any explanation.
+- The architecture is way too old fashioned. To change something takes so much time! For example, mainframe - it is so enormous and old that it cannot be properly tested. The architecture is underdimenzioned. The opposite is an overdimenzioned architecture - to build anything takes too much time.
+-  People are doing a repeptitive work that can be done by machines. The companies do not use [continuous integration](/practices/continuous-integration) - for example, they implement everything to production by hand even though they can automatize the process.
```

Comment:
Maybe change "... work that can be done by machines." on "... work that can be automated."